azext_edge/edge/providers/edge_api/mq.py

```diff
@@ -11,16 +11,8 @@
 class MqResourceKinds(ListableEnum):
     BROKER = "broker"
     BROKER_LISTENER = "brokerlistener"
-    BROKER_DIAGNOSTIC = "brokerdiagnostic"
-    DIAGNOSTIC_SERVICE = "diagnosticservice"
     BROKER_AUTHENTICATION = "brokerauthentication"
     BROKER_AUTHORIZATION = "brokerauthorization"
-    MQTT_BRIDGE_TOPIC_MAP = "mqttbridgetopicmap"
-    MQTT_BRIDGE_CONNECTOR = "mqttbridgeconnector"
-    DATALAKE_CONNECTOR = "datalakeconnector"
```

Comment:
There are some straggler Datalake, kafka, etc mentions but these seem to be support bundle related. Make sure to remove them at some point (make sure not to touch the generated sdk)